narciuch/assignment3/narciuch-3/test/DictionaryTest.java

```diff
@@ -0,0 +1,26 @@
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertTrue;
+
+public class DictionaryTest {
+  String[] words = {"CAR", "CARD", "CART", "CAT"};
+  Dictionary dict = new Dictionary(words);
+
+  @Test
+  public void correctlyIdentifiesWord() {
+    assert(dict.isWord("CAR"));
```

Comment:
I think you meant `assertTrue`, since you already imported it?

`assert` kind of works, but it is better for sanity checks in main code rather than tests.

Reply:
Interesting observation. Will do